Changelogs for Digital Twin Manager Plugins 2.1.4 (#3399)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1021-5-0-Added-support-for-installing-other-official-plugins-within-the-DTM-application.md

---
date: ''

> title: Improved subassets handling with multiple tabs and updated the access path.

Reviewer: ```suggestion
title: Improved subassets handling with multiple tabs and updated the access path
```

> title: Added support for installing other official plugins within the DTM application

Reviewer: The title does not really match the description?

product_area: Application enablement & solutions
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
component:
  - value: component-Tl88RYb4A
    label: Digital Twin Manager
build_artifact:
  - value: tc-wYIY0MBDO
    label: dtm
ticket: CTM-1997
version: 1021.4.0
---

> Previously, integrating Subassets with other tab-based plugins led to action-related errors during tab switching. This has been resolved. Path to access subassets is now changed from /asset/:id/subassets to /group/:id/subassets

Reviewer: ```suggestion
Previously, integrating subassets with other tab-based plugins led to action-related errors during tab switching. This issue has been resolved. The path to access subassets has been changed from `/asset/:id/subassets` to `/group/:id/subassets`. 
```

> Previously, integrating subassets plugin from DTM with other tab-based plugins led to action-related errors during tab switching. This issue has been resolved. The path to access subassets has been changed from `/asset/:id/subassets` to `/group/:id/subassets`. 

Reviewer: ```suggestion
Previously, integrating the subassets plugin from Digital Twin Manager with other tab-based plugins led to errors during tab switching. This issue has been resolved. The path to access subassets has been changed from `/asset/:id/subassets` to `/group/:id/subassets`. 
```
